app/[lang]/_components/LocalizedLink.tsx

```diff
@@ -1,74 +1,162 @@
-'use client';
+'use client'
 
-import Link from 'next/link';
-import {usePathname} from 'next/navigation';
-import {forwardRef} from 'react';
+import Link from 'next/link'
+import {usePathname} from 'next/navigation'
+import {forwardRef} from 'react'
 
-import {DEFAULT_LANGUAGE, getLanguageFromPath} from '@/app/[lang]/_utils/i18nconfig';
+import {DEFAULT_LANGUAGE, getLanguageFromPath} from '@/app/[lang]/_utils/i18nconfig'
 
-import type {LinkProps} from 'next/link';
-import type {AnchorHTMLAttributes, ReactNode} from 'react';
-import type {UrlObject} from 'url';
+import type {LinkProps} from 'next/link'
+import type {AnchorHTMLAttributes, ReactNode} from 'react'
+import type {UrlObject} from 'url'
+
+/* eslint-disable @typescript-eslint/naming-convention */
+declare const __adrsbl: {run: (event: string, conversion: boolean) => void} | undefined
 
 type TLocalizedLinkProps = LinkProps &
 	AnchorHTMLAttributes<HTMLAnchorElement> & {
-		children: ReactNode;
-	};
+		children: ReactNode
+	}
 
 /**
  * A localized version of Next.js Link that automatically prepends the current language
- * to internal links when needed.
```

Comment:
This line is still removed?

Reply:
Ops, just restored the full commentary again. I was suppose to clean some co-pilot things.